### DISCUSSION TITLE
Use Inliner

### DISCUSSION THREAD

**Discussion by Spidy104** [slycelote/caide#42] [Q&A] [answered]
I had a question, caide's inliner is incomparable, but can I just use the caide inliner only, is it possible, if yess, then how??

**Comment by slycelote** [comment, marked as answer]
[caide-cpp-inliner](https://github.com/slycelote/caide-cpp-inliner) contains a simple command-line executable for this (mostly intended for testing purposes), but I don't provide builds at the moment, so you'll have to compile yourself. There are nightly Linux builds on [GitLab](https://gitlab.com/slycelote/caide-cpp-inliner/-/pipelines) that may or not work for you.

EDIT: Fixed the URL.
